singer_sdk/sinks/core.py

```diff
@@ -52,6 +50,9 @@ class Sink(metaclass=abc.ABCMeta):
 
     MAX_SIZE_DEFAULT = 10000
 
+    record_validator = fastjsonschema
+    validate_field_string_format = False
```

Comment:
Let's make sure this shows up in the docs:

```suggestion
    validate_field_string_format = False
    """Enable format validation, for example for `date-time` string fields."""
```

Reply:
Yup, nice!

https://meltano-sdk--2066.org.readthedocs.build/en/2066/classes/singer_sdk.Sink.html#singer_sdk.Sink.validate_field_string_format